pkg/flowaggregator/flowaggregator.go

```diff
@@ -724,3 +904,99 @@ func (fa *flowAggregator) createInfoElementForTemplateSet(ieName string, enterpr
 	}
 	return ie, nil
 }
+
+func (fa *flowAggregator) watchConfiguration(stopCh <-chan struct{}) {
+	klog.Info("Watching for FlowAggregator configuration file")
+	for {
+		select {
+		case <-stopCh:
+			close(fa.updateCh)
+			return
+		case event, ok := <-fa.configWatcher.Events:
+			klog.InfoS("Event happened", "event", event.String())
+			if !ok {
+				// If configWatcher event channel is closed, we kill the flow-aggregator Pod to restore
+				// the channel.
+				klog.Fatal("ConfigWatcher event channel closed")
+			}
+			if err := fa.handleWatcherEvent(); err != nil {
+				// If the watcher cannot add mounted configuration file or the configuration file is not readable,
+				// we kill the flow-aggregator Pod (serious error)
+				klog.Fatalf("Cannot watch or read configMap: %v", err)
+			}
+		case err := <-fa.configWatcher.Errors:
+			if err != nil {
+				// If the error happens to watcher, we kill the flow-aggregator Pod.
+				// watcher might be shut-down or broken in this situation.
+				klog.Fatalf("configWatcher err: %v", err)
+			}
+		}
+	}
+}
+
+func (fa *flowAggregator) handleWatcherEvent() error {
+	data, err := ioutil.ReadFile(fa.configFile)
+	if err != nil {
+		return fmt.Errorf("cannot read FlowAggregator configuration file: %v", err)
+	}
+	opt, err := util.LoadConfig(data)
+	if err != nil {
+		klog.Error(err)
+		return nil
+	}
+	if bytes.Equal(data, fa.configData) {
+		klog.InfoS("Flow-aggregator configuration doesn't changed")
```

Comment:
```suggestion
		klog.InfoS("Flow-aggregator configuration didn't change")
```

I notice that you sometimes use `klog.InfoS` and sometime use `klog.Info` for log messages with no KV pairs. I would recommend being consistent and using `klog.InfoS` everywhere.

Reply:
Thanks for the suggestion. I've changed all klog.info to klog.InfoS